mkosi.conf

```diff
@@ -23,7 +23,7 @@ BuildSourcesEphemeral=yes
 @QemuMem=4G
 @RuntimeSize=8G
 ToolsTreePackages=virtiofsd
-KernelCommandLineExtra=systemd.crash_shell
+@KernelCommandLineExtra=systemd.crash_shell
```

Comment:
Why make this a default, I would hope these are good defaults for the tests as well and we need to override only a few (which we can do by just appending to the list)

Reply:
This did not work for me.

I found that if I added `--kernel-command-line-extra=selinux=1` it would not be enabled because it generated a command line like `... selinux=1 ... selinux=0 apparmor=0 enforcing=0 ...`.

The furthest I got working out why was that in order to process includes on the command-line and config files properly everything was processed in-order, and since the mkosi command-line was parsed before `mkosi.conf` its contents overrode the command-line.